src/pages/projects/[project]/components/ProjectContent/hook/useQuery.ts

```diff
@@ -18,7 +18,9 @@ const useQuery = () => {
   const [project, setProject] = useState<Project>();
 
   async function loadQuery() {
```

Comment:
Pode atualizar para mim?

```suggestion
  async function loadProject() {
```